src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -378,4 +378,7 @@ Modelers:
   <data name="TrackedResourcePatchOperationMissing" xml:space="preserve">
     <value>Tracked resource '{0}' must have patch operation that at least supports the update of tags.</value>
   </data>
+  <data name="ArmPropertiesBagValidationMessage" xml:space="preserve">
+    <value>Properties bag [{0}] should not be repeated for ARM top level properties</value>
```

Comment:
Agreed, @ravbhatnagar the message above would look something like "Properties bag [a, b, c, d] should not be repeated for ARM top level properties" where a,b,c,d are the properties found to be violating the rule. Any suggestions here?